DependencyInjection/CmfCoreExtension.php

```diff
@@ -118,9 +118,6 @@ public function prepend(ContainerBuilder $container)
                         break;
                     case 'cmf_routing':
                         $routePaths = array($persistenceConfig['basepath'].'/routes');
```

Comment:
Hmm, yeah. This is a tricky one, as we would like to change the default to the configured basepath of the CoreBundle, but we don't want to include it when the option is overriden by user config.